boot/pod/src/boot/util.clj

```diff
@@ -57,6 +57,14 @@
       (print ((or color identity) (apply format args)))
       (flush))))
 
+(defmacro print**
+  "Macro version of boot.util/print* but arguments are only evaluated
+  when the message will be printed."
+  [verbosity fmt & args]
+  `(when (>= @*verbosity* ~verbosity)
+     (binding [*out* *err*]
+       (print (ansi/bold-cyan (format ~fmt ~@args))))))
```

Comment:
Uhm, maybe the color part can be improved here (haven't noticed that immediately, maybe cause it's late :smile:)